packages/feedback/src/FeedbackForm.js

```diff
@@ -32,6 +32,7 @@ const FeedbackForm = ({
   onClose,
   faceOptions,
   aboutOptions,
+  customPlaceholder,
```

Comment:
there are multiple placeholders in this component. this prop name should be more specific. perhaps `aboutPlaceholder` ?